### PR TITLE
support downloading server and toolkit packages under arm

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: binary-package.md [hunk 1]
```diff
@@ -5,71 +5,71 @@ summary: 了解 TiDB 离线包及其包含的内容。
 
 # TiDB 离线包
 
-在[使用 TiUP 离线部署 TiDB](/production-deployment-using-tiup.md#离线部署) 前，你需要在[官方下载页面](https://pingcap.com/zh/product#SelectProduct)选择对应版本的 TiDB server 离线镜像包（包含 TiUP 离线组件包）。
+在[使用 TiUP 离线部署 TiDB](/production-deployment-using-tiup.md#离线部署) 前，你需要在[官方下载页面](https://pingcap.com/zh/product#SelectProduct)选择对应版本的 TiDB server 离线镜像包（包含 TiUP 离线组件包）。TiDB 支持 amd64 和 arm64 两种架构的离线包。
 
 TiDB 提供两个二进制离线包：`TiDB-community-server` 软件包和 `TiDB-community-toolkit` 软件包。
```
Comment (qiancai) [inline binary-package.md:10]:
```suggestion
在[使用 TiUP 离线部署 TiDB](/production-deployment-using-tiup.md#离线部署) 前，你需要在[官方下载页面](https://pingcap.com/zh/product#SelectProduct)选择对应版本的 TiDB server 离线镜像包（包含 TiUP 离线组件包）。

TiDB v6.2.0 提供了 amd64 和 arm64 两种架构的离线包。对于每种架构，TiDB 提供了两个二进制离线包：`TiDB-community-server` 软件包和 `TiDB-community-toolkit` 软件包。
```

**Comment by shichun-0415** [comment]
@heipark PTAL

File: download-ecosystem-tools.md [hunk 2]
```diff
@@ -37,16 +37,20 @@ TiDB 工具包中包含了一些常用的 TiDB 工具，例如数据导出工具
...
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)  | `pump-{version}-linux-{arch}.tar.gz` <br/>`drainer-{version}-linux-{arch}.tar.gz` <br/>`binlogctl` <br/>`reparo`  |
+| [Backup & Restore (BR)](/br/backup-and-restore-overview.md)  | `br-{version}-linux-{arch}.tar.gz`  |
 | [sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md)  | `sync_diff_inspector`  |
 | [TiSpark](/tispark-overview.md)  | `tispark-{tispark-version}-any-any.tar.gz` <br/>`spark-{spark-version}-any-any.tar.gz`  |
-| [PD Recover](/pd-recover.md)  | `pd-recover-{version}-linux-amd64.tar.gz` |
+| [PD Recover](/pd-recover.md)  | `pd-recover-{version}-linux-{arch}.tar.gz` |
+
+> **注意：**
+>
+> 以上内容中，`{version}` 变量会被替换为实际的版本号，`{arch}` 变量会被替换为实际的架构（amd64 或 arm64）。
```
Comment (qiancai) [inline download-ecosystem-tools.md:53]:
```suggestion
> 以上离线包名称中，`{version}` 取决于离线包中工具的版本号，`{arch}` 取决于离线包对应的架构（amd64 或 arm64）。
```

**Review by heipark — APPROVED** [review]
LGTM

File: binary-package.md [hunk 3]
```diff
@@ -7,69 +7,77 @@ summary: 了解 TiDB 离线包及其包含的内容。
...
-| playground-{version}-linux-amd64.tar.gz |  |
+| cluster-{version}-linux-{arch}.tar.gz |  |
+| insight-{version}-linux-{arch}.tar.gz |  |
+| diag-{version}-linux-{arch}.tar.gz | 从 v6.0.0 起新增 |
+| influxdb-{version}-linux-{arch}.tar.gz |  |
+| playground-{version}-linux-{arch}.tar.gz |  |
+
+> **注意：**
+>
+> 以上离线包名称中，`{version}` 取决于离线包中工具的版本号，`{arch}` 取决于离线包对应的架构（amd64 或 arm64）。
```
Comment (qiancai) [inline binary-package.md:37]:
```suggestion
> 以上离线包名称中，`{version}` 取决于离线包中内容的版本号，`{arch}` 取决于离线包对应的架构（amd64 或 arm64）。
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**